onnxruntime/test/testdata/transform/fusion/create_conv_clip11.py

```diff
@@ -4,21 +4,24 @@
 
 graph = helper.make_graph(
     [  # nodes
-        # fusable, const_min_negative should be replaced
-        helper.make_node("Conv", ["X", "W"], ["conv0_out"], "Conv0"),
+        # fusable, min/max from constant inputs.
+        helper.make_node("Conv", ["X0", "W"], ["conv0_out"], "Conv0"),
         helper.make_node("Clip", ["conv0_out", "const_min", "const_max"], ["clip0_out"], "Clip0"),
 
         # mutable input. no fusion.
-        helper.make_node("Conv", ["X", "W"], ["conv1_out"], "Conv1"),
+        helper.make_node("Conv", ["X1", "W"], ["conv1_out"], "Conv1"),
         helper.make_node("Clip", ["conv1_out", "mutable_min", "const_max"], ["clip1_out"], "Clip1"),
 
-        # fusabled. default min/max.
-        helper.make_node("Conv", ["X", "W"], ["conv2_out"], "Conv2"),
+        # fusable. default min/max.
+        helper.make_node("Conv", ["X2", "W"], ["conv2_out"], "Conv2"),
         helper.make_node("Clip", ["conv2_out"], ["clip2_out"], "Clip2"),
     ],
-    "ConvClipFusion",  #name
+    "ConvClipFusion",  # name
     [  # inputs
-        helper.make_tensor_value_info('X', TensorProto.FLOAT, [1, 1, 7]),
+
+        helper.make_tensor_value_info('X0', TensorProto.FLOAT, [1, 1, 7]),
+        helper.make_tensor_value_info('X1', TensorProto.FLOAT, [1, 1, 7]),
+        helper.make_tensor_value_info('X2', TensorProto.FLOAT, [1, 1, 7]),
```

Comment:
using distinct inputs for Convs otherwise the common subexpression elimination transformer folds them

Reply:
nit: in general, if it's worth adding a comment explaining something in the PR it's often good to add a comment in the code as well.